projects/plugins/jetpack/to-test.md

```diff
@@ -1,6 +1,4 @@
-## Jetpack 14.1
-
-### Before you start:
```

Comment:
This bit seems unexpected.

Reply:
I removed it since I didn't see any value in keeping the version number there: the version with the testing instructions (whatever it might be) should be the one tested.

Reply:
It could help to know which version the instructions are written for. If it's still showing an old version, that probably indicates someone didn't blank them after the release.

Reply:
🤷 4671f01afb